src/data_ingest/history_loader.py

```diff
@@ -0,0 +1,48 @@
+"""Functions relating to loading historic data"""
+from typing import Generator, List
+import logging
+import re
+import os
+
+history_loader_logger = logging.getLogger(__name__)
+
+
+def history_years(current: int, back_history: int) -> Generator[int, None, None]:
+    """Gets the year or years to load historic data for"""
+    if back_history > 0:
+        # create a generator to yield the years to load
+        for i in range(back_history):
+            yield current - 1
+            current -= 1
+    else:
+        print("No historic data to load")
+        return None
+
+
+def hist_paths_to_load(hist_folder: str, history_years: Generator[int, None, None]) -> List[str]:
+    """Creates a list of paths to load historic data from"""
+    # Create a list of paths to load
+    hist_paths = []
+    for year in history_years:
+        hist_path = hist_folder + "qv_BERD_" + str(year) + "12_qv6_reformatted.csv"
+        hist_paths.append(hist_path)
+    return hist_paths
+
+
+def load_history(year_generator: Generator[int, None, None], hist_folder_path: str, read_csv_func: callable) -> None:
```

Comment:
When you ran the pipeline, it logged which years were loaded. My run just outputted a message that showed that loading was complete. I didn't see how many files were loaded and which ones they were. Am i missing something?

Reply:
I get this when I run the pipeline. 

![image](https://github.com/ONSdigital/research-and-development/assets/67740306/84ebd5f5-6b0c-4a0d-aa70-bfef07711922)


Do you think there's a need to log anything else?

Reply:
Yes I have pushed that change now so you should get the same.